src/ConfigurationExtensions.cs

```diff
@@ -33,5 +34,24 @@ public static KormConnectionSettings GetKormConnectionString(this IConfiguration
             }
             return null;
         }
+
+        private static string SetApplicationName(string connectionString, string applicationName)
+        {
+            try
+            {
+                var builder = new SqlConnectionStringBuilder(connectionString);
+
+                if (string.IsNullOrEmpty(builder.ApplicationName) && !string.IsNullOrEmpty(applicationName))
```

Comment:
Ja by som tú podmienku dal na začiatku. Načo parsovať a vytvárať `SqlConnectionStringBuilder`, keď vlastne sa to vôbec možno nepoužije?

Reply:
Pull request uzatváram. Riešenie v PR nie je dobré, pretože application name by sa pridal do connection stringu len v prípade, že korm databáza sa konfiguruje pomocou extension metódy (https://github.com/Kros-sk/Kros.KORM.Extensions.Asp/blob/fd42fd1bf3aa20c3712c707b9d194a1adcf79c37/src/ServiceCollectionExtensions.cs#L57), v ktorej sa volá metóda GetKormConnectionString. Korm databáza sa dá nakonfigurovať aj pomocou iných extension metód, prípadne úplne bez použitia extension metód z Kros.KORM.Extensions.Asp. Pridanie názvu aplikácie by sa teda malo robiť priamo v Kros.Korm knižnici kde sa používa connection string z KormConnectionSettings.